website/blog/2020-11-19-2.2.0.md

```diff
@@ -0,0 +1,1135 @@
+---
+author: "Sosuke Suzuki (@sosukesuzuki)"
+authorURL: "https://github.com/sosukesuzuki"
+title: "Prettier 2.2: new JavaScript parsers, TS 4.1 and ESM standalone bundles"
+---
+
+This release supports new JavaScript parsers [espree](https://github.com/eslint/espree) and [meriyah](https://github.com/meriyah/meriyah), supports [TypeScript 4.1](https://devblogs.microsoft.com/typescript/announcing-typescript-4-1-rc/), ships ESM standalone bundles for modern browsers, and includes many bug fixes and improvements!
+
+<!--truncate-->
+
+## Highlights
+
+### JavaScript
```

Comment:
I think we can remove categories for Highlights?

Reply:
I prefer current structure.